doc/source/dsintro.rst

```diff
@@ -366,13 +366,32 @@ dictionary.
 
 .. _basics.dataframe.from_series:
 
-From a Series
-~~~~~~~~~~~~~
+From a Panda Series
+~~~~~~~~~~~~~~~~~~~
 
 The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+.. ipython:: python
+
+   df = pd.DataFrame()
```

Comment:
We typically don't want to encourage creating an empty dataframe first, and then filling it. 

You can give an example of directly creating a DataFrame from a Series like:

```
s = pd.Series([1, 2, 3])
pd.DataFrame(s)
```